src/Behat/Behat/Definition/Translator/DefinitionTranslator.php

```diff
@@ -39,8 +44,15 @@ public function translateDefinition(Suite $suite, Definition $definition, $langu
     {
         $assetsId = $suite->getName();
         $pattern = $definition->getPattern();
+        $cacheKey = sprintf('%s|%s|%s', $assetsId, $pattern, $language ?? '');
```

Comment:
this should use `$this->translator->getLocale()` as default locale, not the empty string, so that an explicit locale and the usage of the default locale with the same locale share the same cache key.

Reply:
Good shout, updated